src/ripple/app/ledger/LedgerToJson.h

```diff
@@ -69,10 +69,13 @@ Json::Value getJson (LedgerFill const&);
 template <typename Object>
 void fillJson (Object& json, LedgerFill const& fill)
 {
+    using namespace ripple::RPC;
+
     auto const& ledger = fill.ledger;
 
     bool const bFull (fill.options & LEDGER_JSON_FULL);
     bool const bExpand (fill.options & LEDGER_JSON_EXPAND);
+    bool const bBinary (fill.options & LEDGER_JSON_BINARY);
```

Comment:
fill.options is an int, but it is used like a std::bitset. Don't need to change in this PR.